### PR TITLE
React practice

### PR DESCRIPTION
**Pull request by Cat20-17**

### CODE REVIEW

File: react-practice/src/components/Accordion/Accordion.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React, {useState} from 'react';
+
+import styles from './Accordion.module.scss';
+import {ReactComponent as ChevronSvg} from '../../images/chevron.svg';
+import classNames from "classnames";
+
+const Accordion = ({ name, content }) => {
+  const [state, setState] = useState(false);
```
Comment (NesQ1kx) [inline react-practice/src/components/Accordion/Accordion.jsx:8]: `[opened, toggleState]`

File: react-practice/src/components/Filter/Filter.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import styles from './Filter.module.scss';
+import Checkbox from "../Checkbox/Checkbox";
+import FilterTitle from "../FilterTitle/FilterTitle";
+
+const Filter = ({ name, options, selectedOptionIds, onChange }) => {
+  const toggleAll = (newChecked) => {
+    console.log(newChecked);
```
Comment (NesQ1kx) [inline react-practice/src/components/Filter/Filter.jsx:9]: Remove

File: react-practice/src/components/Accordion/Accordion.jsx (hunk 1 above)
Comment (Cat20-17) [reply react-practice/src/components/Accordion/Accordion.jsx]: done

File: react-practice/src/components/Filter/Filter.jsx (hunk 2 above)
Comment (Cat20-17) [reply react-practice/src/components/Filter/Filter.jsx]: done

**Review by NesQ1kx — APPROVED** [review]

**Pull request merged**